sdks/java/extensions/ml/build.gradle

```diff
@@ -47,6 +53,12 @@ dependencies {
     testCompile project(path: ':sdks:java:core', configuration: 'shadowTest')
     compile 'com.google.cloud:google-cloud-vision:1.99.3'
     testCompile library.java.mockito_core
+    testCompile group: 'com.google.http-client', name: 'google-http-client', version: '1.38.0'
+    testCompile group: 'com.google.protobuf', name: 'protobuf-java-util', version: '3.14.0'
```

Comment:
also `library.java.protobuf_java_util`

you can find these at https://github.com/apache/beam/blob/master/buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy